src/attributes/model.jl

```diff
@@ -66,3 +66,21 @@ function MOI.set(
 
     return nothing
 end
+
+function MOI.get(model::Virtual.Model, ::MOI.ListOfVariableAttributesSet)
+    list = MOI.get(model.source_model, MOI.ListOfVariableAttributesSet())
```

Comment:
Do we want to start generating documentation on functions to have automatic docs from those?